Hazel/SendError.cs

```diff
@@ -0,0 +1,15 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+
+namespace Hazel
+{
+    [Flags]
+    public enum SendError
```

Comment:
Consider rename to `SendErrors` for plural enum naming convention, but I'd say also OK to leave as is